src/ui/ui.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <vector>
+#include "../image_buffer.h"
+#include "../math/math_base.h"
+
+class UiWindow
+{
+public:
+    UiWindow( const PenguinV_Image::Image & image = PenguinV_Image::Image() );
+    virtual ~UiWindow();
+    void show(); // show window at the screen
+    void setImage( const PenguinV_Image::Image & image ); // replaces existing shown image by new image
+    virtual void drawPoint( const Point2d & point );
+    virtual void drawLine( const Point2d & start, const Point2d & end );
+protected:
+    virtual void draw();
```

Comment:
No-no, no need pure virtual. My idea is to make base class non-abstract so in case of no UI support (noQT for example) such class does nothing but code still compiles. What do you think?

Reply:
I see your point. I agree.

Reply:
I'll rename it to `_display`. Symbol `_` put function name at the end of function list :)